services/compute/virtualmachine/client.go

```diff
@@ -137,7 +137,7 @@ func (c *VirtualMachineClient) Update(ctx context.Context, group string, vmName
 }
 
 // Resize the Virtual Machine
-func (c *VirtualMachineClient) Resize(ctx context.Context, group string, vmName string, newSize compute.VirtualMachineSizeTypes, newCustomSize *compute.VirtualMachineCustomSize) (err error) {
+func (c *VirtualMachineClient) Resize(ctx context.Context, group string, vmName string, hardware *compute.HardwareProfile) (err error) {
```

Comment:
Won't this be a breaking change?

Reply:
Why not just add one more param to pass in list of GPUs instead of passing in h/w profile. I guess changing dynamic memory configuration is not supported

Reply:
Oh understood, yes will just add a new param for gpu

Reply:
Done